concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+It should start with a brief discussion about the team’s goals. This should be an open, but high-level, discussion that will help frame the kinds of issues we should prioritize. It shouldn’t take long -- no more than 15-20 minutes.
+
+With the context of that discussion in mind, we’ll move to backlog review. As detailed above, our backlogs are contained in individual boards, and we’ll run through each one as part of the planning process.
+
+For each board:
+- Briefly run through the Backlog column to decide if there are any issues we want to promote to "Tracking" status.
+- Run through all items in "Tracking" and decide which ones are highest priority, and who will work on them. They will become the issue’s assignee.
+
+From there, automation will aid the process -- any newly assigned issues will:
+- be moved to the Active column in the backlog board
+- automatically added to the Assigned column of the Active Sprint board
+
+This process repeats for every board, after which our Active Sprint board will be populated with our workload for the next two weeks, and our sprint planning is complete.
+
+### Daily Standup
+Our daily standups will take place every day we don’t have a longer meeting (such as sprint planning/retro) for half an hour. This is meant to be a quick check-in to ensure newly-opened issues get quick responses and blockers get resolved.
+
+First, if there are any new people on the call, we'll welcome them and allow them to ask questions or have discussions as needed.
+
+Next, we’ll have a look at all untriaged issues -- any that do not have a label, and are not part of a project. This will allow the whole team to discuss them and ensure they get moved to the appropriate backlog board and labeled correctly. This shouldn’t take more than 15 minutes, though that’s a soft limit -- if a good conversation is happening, it shouldn’t be cut off.
+
+Finally, if there's time, we'll dedicate some time to a quick review of our Current Sprint board. This will serve as an opportunity for developers to elaborate on any blockers and ask for reviews or re-reviews as needed. We should strive to do this every day if possible, but if prior discussion makes that infeasible, we don't _have_ to do it (there's always tomorrow). Project discussion is more important.
+
+### Sprint Retrospective
+Our final meeting is the sprint retrospective. This should be a quick half-hour meeting that takes the place of Monday’s standup where we discuss:
+- What we accomplished in the last sprint (perhaps with a demo, if appropriate)
+- What didn’t get done, and why (blockers? More complex than anticipated? etc)
+
+If there is leftover work on our Active Sprint board, we should decide whether we should continue working on it or demote it to a backlog board. Then, we move onto the next sprint and do it all over again.
```

Comment:
We should discuss why it didn't get completed. This might happen naturally as whoever is working on it explains blockers throughout the week, but broadly speaking understanding the consequences of our planning will help us plan better moving forward. There's a good chance we underestimated the difficulty, or perhaps there was a lot of scaffolding work that needed to fall into place before this one did that we didn't foresee. In both cases, we should regroom the issue and further decompose it if necessary.

Reply:
"We" is dangerous here.  Again - there needs to be a discussion of roles here.  Until we have the roles described, we're going to struggle with what happens in each type of meeting.

Reply:
Agree with the above comments: without clear roles and responsibilities it is easy to repeat the story of [Everybody, Anybody, Somebody, and Nobody](https://www.lollydaskal.com/leadership/story-everybody-somebody-anybody-nobody/) or just talk in circles without reaching an output. I've seen that meetings tend to be ineffective without understood inputs, agendas/roles, and outputs. In this case, the outputs should also include how to improve the process itself (as mentioned below), how to address the issues experienced during the sprint, and the personnel responsible for each output task.

Reply:
Agreed. I want roles clearly defined.